apps/condo/bin/fix-tickets-client-name-client-phone.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { find, getByCondition } = require('@condo/keystone/schema')
+const isEmpty = require('lodash/isEmpty')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+
+class FixTicketsClientNameClientPhone {
+    context = null
+    brokenTickets = []
+
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+        this.context = await keystone.createContext({ skipAccessControl: true })
+    }
+
+    async findBrokenTickets () {
+        // Tickets from resident with contact, but without clientName and clientPhone
+        this.brokenTickets = await find('Ticket', {
```

Comment:
if there would be lots of tickets > 100, this will fail. we have `loadListByChunks` for such cases

Reply:
number of tickets - 126

Reply:
But you can run the script 2 times :)